scripts/release-tasks.ts

```diff
@@ -2,15 +2,8 @@ import color from 'ansi-colors';
 import Listr, { ListrTask } from 'listr';
 
 import { bundleBuild } from './build';
-import { validateBuild } from './test/validate-build';
 import { BuildOptions } from './utils/options';
-import {
-  isPrereleaseVersion,
-  isValidVersionInput,
-  postGithubRelease,
```

Comment:
`postGithubRelease` is still in the codebase, but technically no longer used. Leaving it here for the moment to see if I can get GitHub to do this for us